src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions.mdx

```diff
@@ -0,0 +1,342 @@
+---
+title: Create and edit alert conditions in the UI
+tags:
+  - Alerts and applied intelligence
+  - Alerts
+  - Alert conditions
+translate:
+  - jp
+metaDescription: "Use the conditions page to identify what triggers an alert policy's notification, starting with the product and type of metric or service."
+redirects:
+  - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/
+  - /docs/alerts/alert-policies/configuring-alerts/managing-your-alerts 
+freshnessValidatedDate: never
+---
+import alertsSetaQuery from 'images/alerts_screenshot-crop_set-a-query.webp'
+
+import alertsFineTuneAlertSignals from 'images/alerts_screenshot-crop_fine-tune-alert-signals.webp'
+
+import alertsSetaThresholdforanAlertCondition from 'images/alerts_screenshot-crop_set-a-threshold-for-an-alert-condition.webp'
+
+import alertsNameYourAlertCondition from 'images/alerts_screenshot-crop_name-your-alert-condition.webp'
+
+import alertsAlertConditionPage from 'images/alerts_screenshot-crop_alert-condition-page.webp'
+
+New Relic alerts are built with alert conditions. If you want to be notified of any unusual behavior in your stack, you'll first need to create an alert condition. An alert condition is essentially a container that you create to define what conditions need to be met before you're notified of any unusual behavior. 
```

Comment:
```suggestion
Alert conditions can elevate the quality of your alerts. An alert condition contains certain parameters or thresholds that need to be met before you're notified of any unusual behavior. They can mitigate excessive alerting, or let your team know when a new or unusual behavior appears
```

I actually think you can write something better but this is just an example of making it more active/direct w/ a story.